### PR TITLE
Create prism-v3.md

### PR DESCRIPTION
**Pull request by XVincentX**

### CODE REVIEW

File: netlify/blog-posts/prism-v3.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  robots: 'index, follow'
+  title: Prism has now a version 3: it's faster and goes Open Source | Stoplight API Corner
+  image: /images/dayne-topkin-cxruh7pmmsu-unsplash.jpg
+  twitter:
+    description: Prism has now a version 3: it's faster and goes Open Source
+    title: Prism has now a version 3: it's faster and goes Open Source | Stoplight API Corner
+    image: /images/dayne-topkin-cxruh7pmmsu-unsplash.jpg
+    username: '@stoplightio'
+---
+# Prism v3 Adds OpenAPI 3.0 and Goes Open Source
```
Comment (tbarn) [inline netlify/blog-posts/prism-v3.md]: Why not make this the title?

File: netlify/blog-posts/prism-v3.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      title: Get Started with Prism
+  enabled: true
+  text: Do you want a mock server on your API descriptions right now?
+meta:
+  description: Prism has now a version 3: it's faster and goes Open Source
+  robots: 'index, follow'
+  title: Prism has now a version 3: it's faster and goes Open Source | Stoplight API Corner
+  image: /images/dayne-topkin-cxruh7pmmsu-unsplash.jpg
+  twitter:
+    description: Prism has now a version 3: it's faster and goes Open Source
```
Comment (tbarn) [inline netlify/blog-posts/prism-v3.md]: Description and title need to be different things or it shows up weird in cards that pull from the metadata.

**Comment by XVincentX** [comment]
@lottamus I can't see the blog post published — is it because I set the date in the future?

**Comment by tbarn** [comment]
@lottamus is looking into why the build is 404ing since I don't have access to the Netlify build system anymore.

**Comment by lottamus** [comment]
@XVincentX looks like invalid YAML

image

**Comment by lottamus** [comment]
You can reproduce by running `yarn build` locally

**Comment by XVincentX** [comment]
gosh, ok thanks!

**Comment by tbarn** [comment]
List of todos (I can help if needed, lmk): 

- [ ] Needs new main image since we used that one for Spectral, maybe a.... prism? 
- [ ] Make sure metadata for title and description are not the same, and that title matches blog post title
- [ ] Link to landing page in the first few paragraphs somewhere
- [ ] Add mention of webinar (see Spectral post for example)

**Comment by XVincentX** [comment]
@tbarn I should have addressed all the outstanding points

**Pull request merged**